src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,101 @@
 package core.basesyntax;
 
+import java.util.Arrays;
```

Comment:
According to the task checklist, you should not use Arrays or any other util class. In your `grow()` method, you use `Arrays.copyOf()`. This is a violation of the task requirements. Instead of using `Arrays.copyOf()`, you should create a new array and copy the elements manually using `System.arraycopy()`. This will ensure that you meet the task requirements.